tests/gui/models/selection-state.spec.js

```diff
@@ -485,6 +487,41 @@ describe('Browser: SelectionState', function() {
           SelectionStateModel.removeImage();
         });
 
+        it('should de-select a previously selected source drive', function() {
+          const imagePath = _.attempt(() => {
+            if (process.platform === 'win32') {
+              return 'E:\\bar\\foo.img';
+            }
+
+            return '/mnt/bar/foo.img';
+          });
```

Comment:
It's a shame this differs slightly from the similar function in `tests/gui/models/drives.spec.js`